ports/nrf/boards/makerdiary_m60_keyboard/mpconfigboard.h

```diff
@@ -40,6 +40,7 @@
 #define MICROPY_QSPI_DATA3                NRF_GPIO_PIN_MAP(1, 12)
 #define MICROPY_QSPI_SCK                  NRF_GPIO_PIN_MAP(1, 11)
 #define MICROPY_QSPI_CS                   NRF_GPIO_PIN_MAP(1, 13)
+#define MICROPY_QSPI_OFF_WHEN_SLEEP
```

Comment:
For newly introduced macros controlling compilation of optional features, we __PREFER__ that you create a macro which is defined to (1) when the optional feature is enabled, and (0) when it is disabled.

For instance in (top level)/py/circpy_mpconfig.mk, add
```
CIRCUITPY_QSPI_OFF_WHEN_SLEEP ?= 0
CFLAGS += -DCIRCPY_QSPI_OFF_WHEN_SLEEP=$(CIRCUITPY_QSPI_OFF_WHEN_SLEEP)
```
in ports/nrf/boards/makerdiary_m60_keyboard/mpconfigboard.h , add
```
CIRCPY_QSPI_OFF_WHEN_SLEEP = 1
```

Then, write
```
#if CIRCPY_QSPI_OFF_WHEN_SLEEP
....
#else
....
#endif
```

One reason we prefer this is because the compiler will give a diagnostic if CIRCPY_QSPI_OFF_WHEN_SLEEP is not defined, e.g., because of a typo.

Reply:
Got it. Will make a change

Reply:
It is a feature for nRF52840. I'm not sure if we put a macro `CIRCUITPY_QSPI_OFF_WHEN_SLEEP` in `py/circpy_mpconfig.mk`

Reply:
@tannewt can you advise what we should do here?

Reply:
I wouldn't do a macro. Just always do it because it's an improvement for all boards.

Reply:
Before enabling it for all nrf52840 boards, I think we should leave some time to test and make sure it doesn't cause any problem.

Reply:
It's fine to test on everything. That way it'll be tested more. The releases we make will be marked unstable so it's ok if it doesn't fully work.

Reply:
OK